doc/nrf/device_guides/working_with_nrf/nrf70/developing/powersave.rst

```diff
@@ -414,6 +417,10 @@ The following shell commands and network management APIs are provided for power
      - wifi twt teardown_all
      - Tear down all sessions
      - TWT operation TWT teardown all flows
+   * - net_mgmt(NET_EVENT_WIFI_TWT_SLEEP_STATE)
```

Comment:
Shouldn't this be net_mgmt_event_notify_with_info(NET_EVENT_WIFI_TWT_SLEEP_STATE) ?